### PR TITLE
Fixed npm command in windows

### LINKED ISSUE #15: Add Windows support
Currently, the `npm` command is not found on windows.

### PR DESCRIPTION
**Pull request by DylanArnold**
Fixes #15

### CODE REVIEW

File: sbt-scalajs-bundler/src/main/scala/scalajsbundler/Commands.scala [hunk 1]
```diff
@@ -3,6 +3,10 @@ package scalajsbundler
 import sbt._
 
 object Commands {
+  private val npm = sys.props("os.name").toLowerCase match {
+    case os if os.contains("win") ⇒ "cmd /c npm"
```
Comment (julienrf) [inline sbt-scalajs-bundler/src/main/scala/scalajsbundler/Commands.scala]: Can you explain what does that mean?

Comment (DylanArnold) [inline sbt-scalajs-bundler/src/main/scala/scalajsbundler/Commands.scala]: It's testing what the current operating system is. Then it uses the built in `cmd` executable in windows to execute the `npm` batch file.

I don't really know why but it seems as though executing a shell command doesn't pick up windows batch files, only executables.

So if it's windows it executes `cmd /c npm update` etc, if it's some other os, just `npm update`. (See the usage of the npm val in string interpolation below).

I got the sys.props trick by looking at the sbt-native-packager source code.

Not really certain if there is better approach but this seems to work.

**Comment by DylanArnold** [comment]
I don't think you need to update the usage of the node command because it is a windows executable rather than a batch file.

So far this seems to be working. This is my first time using scalajs-bundler so I haven't completely got a working project yet, however I noticed that the bundler js files are being created, so everything appears fine now.

File: sbt-scalajs-bundler/src/main/scala/scalajsbundler/Commands.scala (hunk 1 above)
Comment (DylanArnold) [inline sbt-scalajs-bundler/src/main/scala/scalajsbundler/Commands.scala]: Oops I just noticed my unicode arrows ⇒. I can update those to => ?

**Comment by DylanArnold** [comment]
What's needed to test the node command?

**Comment by julienrf** [comment]
Use the last version and run the `fastOptJS::webpack` sbt command.

**Comment by julienrf** [comment]
OK, thanks again @DylanArnold for your work on this, can you confirm me that you can run the `fastOptJS::webpack` command with the last version? Then I think this is good to merge!

**Comment by DylanArnold** [comment]
@julienrf No problem. Thanks for all the hard work you put into this project :) 

I just tested `fastOptJS::webpack` and it executes with success.

**Comment by julienrf** [comment]
Great!

**Pull request merged**